apps/sap-commerce-cloud/frontend/src/components/Field/Field.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import { FieldAppSDK } from '@contentful/app-sdk';
+import Field from './Field';
+import { render } from '@testing-library/react';
+import { AppParameters } from '../../interfaces';
+import { vi } from 'vitest';
+
+describe('Field', () => {
+  it('should render', () => {
+    const { container } = render(
+      <Field
+        {...{
+          sdk: {
```

Comment:
In a couple of these components it looks like you are creating a mock sdk to pass as a prop. We already have a `mockSdk` that can be used here: https://github.com/contentful/apps/blob/feat/sap-app-refactor-ext5483/apps/sap-commerce-cloud/frontend/src/__mocks__/mockSdk.ts. Maybe to reduce some duplicative code you could update this mock and use this instead?

Reply:
will do 👍